devel/leiningen/Portfile

```diff
@@ -5,6 +5,7 @@ PortGroup           java 1.0
 
 name                leiningen
 version             2.10.0
+revision            1
```

Comment:
No need to bump the revision, as nothing in the build or installed files changed. You're just fixing a new build error caused by a change in base.